### PR TITLE
Fixed rowCount for TYPO3 8.7

### PR DESCRIPTION
**Pull request by vandres**

### CODE REVIEW

**Comment by oaltmann** [comment]
Thanks for the php bug fix! But i still can't login. The same problem as I have already described. After the request from the openID Server I am back in the login area with this get parameters:

> index.php?login_status=login&tx_openid_location=http%3A%2F%2Ft3res87.w15d.tankom.de%2Ftypo3%2F%3FloginProvider%3D1433416748&tx_openid_location_signature=261c1b0bcc28fed56ffc91e0b53b97e438627ead&tx_openid_mode=finish&tx_openid_claimed=https%3A%2F%2Foaltmann.id.tankom.de%2F&tx_openid_signature=069e609d05e23adfd31f13a73bab2848a1d09f8b&janrain_nonce=2017-04-13T13%3A51%3A27Z9ZtqPW&openid.assoc_handle=%7BHMAC-SHA1%7D%7B58ef825f%7D%7BxBoXyg%3D%3D%7D&openid.claimed_id=https%3A%2F%2Foaltmann.id.tankom.de%2F&openid.identity=https%3A%2F%2Foaltmann.id.tankom.de%2F&openid.mode=id_res&openid.ns=http%3A%2F%2Fspecs.openid.net%2Fauth%2F2.0&openid.op_endpoint=https%3A%2F%2Fid.tankom.de%2Fopenid%2Fprovider&openid.response_nonce=2017-04-13T13%3A51%3A32ZsBaJf7&openid.return_to=http%3A%2F%2Ft3res87.w15d.tankom.de%2Ftypo3%2Findex.php%3Flogin_status%3Dlogin%26tx_openid_location%3Dhttp%253A%252F%252Ft3res87.w15d.tankom.de%252Ftypo3%252F%253FloginProvider%253D1433416748%26tx_openid_location_signature%3D261c1b0bcc28fed56ffc91e0b53b97e438627ead%26tx_openid_mode%3Dfinish%26tx_openid_claimed%3Dhttps%253A%252F%252Foaltmann.id.tankom.de%252F%26tx_openid_signature%3D069e609d05e23adfd31f13a73bab2848a1d09f8b%26janrain_nonce%3D2017-04-13T13%253A51%253A27Z9ZtqPW&openid.sig=losW6ZsQYGkzGnQ1kEfjEbMU5zY%3D&openid.signed=assoc_handle%2Cclaimed_id%2Cidentity%2Cmode%2Cns%2Cop_endpoint%2Cresponse_nonce%2Creturn_to%2Csigned

Is it a php7 Problem? On php5.6 does it work fine.

**Comment by vandres** [comment]
@oaltmann 
I am working with PHP7, too. I have this bug sometimes when working with Docker. For me one of the following steps helps most of the time:

- install php70-gmp
- recreate the Docker-Image
- delete all Browser-Cookies/Caches
- restart Browser
- restart PC/Mac

I am not having this bug, when working on a remote host with PHP7.

File: ext_emconf.php [hunk 1]
```diff
@@ -8,12 +8,12 @@
     'uploadfolder' => 0,
     'createDirs' => '',
     'clearCacheOnLoad' => 0,
-    'version' => '8.0.0',
```
Comment (liayn) [inline ext_emconf.php:11]: please do not change the version number in casual fixes. will do that in dedicated release commits. Thanks.

**Comment by liayn** [comment]
@vandres Can you please repush your commit without the version number adjustments and also with a commit message conforming to the Core rules?
https://docs.typo3.org/typo3cms/ContributionWorkflowGuide/Appendix/GeneralTopics/CommitMessage.html

Specifically the classification is missing and the time form is the wrong one.
Thanks.

**Comment by vandres** [comment]
I've created a separate pull request

**Pull request closed without merging**